### DISCUSSION TITLE
--set-file equivalent helmrelease ?

### DISCUSSION THREAD

**Discussion by samos667** [fluxcd/flux2#3222] [Q&A] [answered]
Hi, i try to deploy stash with flux but in have a problem for translate HELM command in my values.yaml

https://stash.run/docs/v2022.09.29/setup/install/community/
In the doc the HELM command to deploy stash is:

helm install stash appscode/stash          \
  --version v2022.09.29                \
  --namespace kube-system                     \
  --set features.community=true               \
  --set-file global.license=/path/to/the/license.txt

This is the helm release:
![image](https://user-images.githubusercontent.com/50653464/196620874-697ec034-1f7c-4cc0-896c-08e4a6a9cc9a.png)

And the configmap:
![image](https://user-images.githubusercontent.com/50653464/196621022-3a3ccf71-c765-4767-a435-91a70191a0e4.png)

I dont know how to proceed for this option " --set-file global.license=/path/to/the/license.txt", what i have tried in my configmap doesn't work. Any idea ?

**Comment by hiddeco** [comment, marked as answer]
You need to omit the `license.txt` from the `.global.license` value, and use the certificate directly as the value. Rough example:

```yaml
global:
  license: |-
    -----BEGIN CERTIFICATE-----
    ...
```

**Reply by samos667** [reply]
Thanks a lot !
